packages/celotool/src/lib/attestation.ts

```diff
@@ -255,9 +274,8 @@ export async function createPhoneNumber(
 
     await twilioClient.incomingPhoneNumbers.create({
       phoneNumber: usableNumber,
-      addressSid,
```

Comment:
Remind me again why wouldn't we try to get phone numbers in other jurisdictions that require the addressSid?

Reply:
i'm unclear what this was originally used for?

Reply:
From trying the API, this parameter has no effect here

Reply:
There seems to be no way AFAICT of creating a number inside a messaging service.  Passing the address in here via `messagingServiceSid` here results in `address not found` for me.

Reply:
So I resolved we would not use a messaging service for the bot, as previously, and just make sure we only select numbers to get attested that definitely can accept incoming messages